website/docs/cloud-docs/users-teams-organizations/api-tokens.mdx

```diff
@@ -49,6 +49,16 @@ Organization API tokens have permissions across the entire organization. They ca
 
 -> **Note:** This endpoint cannot be accessed with [organization tokens](#organization-api-tokens). You must access it with a [user token](/terraform/cloud-docs/users-teams-organizations/users#api-tokens) or [team token](#team-api-tokens).
 
+## Audit Trails Tokens
```

Comment:
To make the plural-singular agree here, we need to either have "trails" or "tokens" be plural, so I suggest keeping "trails" singular:
```suggestion
## Audit trail tokens
```